sycl/test-e2e/lit.cfg.py

```diff
@@ -135,10 +135,15 @@
 if lit_config.params.get('matrix-fp16', False):
     config.available_features.add('matrix-fp16')
 
-#support for LIT parameter ze_debug<num>
-if lit_config.params.get('ze_debug'):
-    config.ze_debug = lit_config.params.get('ze_debug')
-    lit_config.note("ZE_DEBUG: "+config.ze_debug)
+#support for LIT parameter ur_l0_debug<num>
+if lit_config.params.get('ur_l0_debug'):
+    config.ur_l0_debug = lit_config.params.get('ur_l0_debug')
+    lit_config.note("UR_L0_DEBUG: "+config.ur_l0_debug)
```

Comment:
I don't think we need this, because it would result in the same issue we're trying to fix.

Reply:
thansk @aelovikov-intel . I'm not familiar with this code, so I will remove it if wanted.

@smaslov-intel , @EwanC : what do you think?

Reply:
I think it is good to have the ability to request debug traces with an option

Reply:
I don't really have an opinion on this. Presumably a user could pass `lit --param extra_environment="UR_L0_DEBUG=1"` to request debug traces regardless, but I don't object to having a convenience option.